roles/docker_image_build/templates/base/entrypoint.sh

```diff
@@ -1,2 +1,11 @@
 #!/usr/bin/env bash
+{{ splunk_home }}/bin/splunk migrate migrate-kvstore-36-40
```

Comment:
This does not cause any issues if mongo is already updated. It also does not cause issues in version 8.2.x where that command is not available.